nilearn/_utils/testing.py

```diff
@@ -186,14 +186,15 @@ def __call__(self, *args, **kwargs):
             basename = os.path.basename(fname)
             if basename in self.csv_files:
                 array = self.csv_files[basename]
+
                 # np.savetxt does not have a header argument for numpy 1.6
```

Comment:
Actually I just checked and I misread the comment: there is no header argument in np.savetxt for numpy 1.6, which by the way is probably the older version we are testing on travis-ci.